granulate_utils/metadata/databricks_client.py

```diff
@@ -44,45 +50,81 @@ def get_webui_address() -> Optional[str]:
     def get_job_name(self) -> Optional[str]:
         # Retry in case of a connection error, as the metrics server might not be up yet.
         for i in range(MAX_RETRIES):
-            time.sleep(10)
             try:
-                return self._get_job_name_impl()
-            except Exception:
-                self.logger.exception("Got Exception while collecting Databricks job name.")
+                if cluster_metadata := self._cluster_all_tags_metadata():
+                    # Got the job name, no need to retry.
+                    name = self._get_name_from_metadata(cluster_metadata)
+                    if name:
+                        self.logger.debug("Found name in metadata.", job_name=name, cluster_metadata=cluster_metadata)
```

Comment:
Let's remove all `.` from log message ends - it's not customary

Reply:
a90a0c8